### ISSUE TITLE
Some remote logging tests are flaky

### ISSUE THREAD

**Issue by enkore** [borgbackup/borg#1081]
https://travis-ci.org/borgbackup/borg/builds/132057852

```py
___________ RemoteRepositoryLoggerTestCase.test_old_format_messages ____________

self = <borg.testsuite.repository.RemoteRepositoryLoggerTestCase testMethod=test_old_format_messages>

    def test_old_format_messages(self):
        self.handler.setLevel(logging.DEBUG)
        logging.getLogger().setLevel(logging.DEBUG)
        self.create_repository("$LOG INFO Remote: old format message")
>       self.assert_equal(self.stream.getvalue(), 'Remote: old format message\n')
E       AssertionError: '' != 'Remote: old format message\n'
E       + Remote: old format message
py34/lib/python3.4/site-packages/borg/testsuite/repository.py:444: AssertionError

======== 1 failed, 532 passed, 141 skipped, 2 xpassed in 469.69 seconds ========
```

This one popped up yesterday once or twice in Travis. Restarting the build helps, so definitely flaky. Could be related to the environmental pollution some other tests do; we should work on cleaning that up, then this might vanish all by itself.

**Comment by edgewood** [comment]
The setup code for those test cases is perhaps more complicated than is necessary.  It runs a remote python command line to inject the log message without changing RemoteRepository.call_many().  I wrote it in the process of exploring how to deal with remote logging, and never revisited it.

I could break out the code in RemoteRepository starting under "if line.startswith('$LOG '):" (or maybe all the code in the enclosing for loop) into a separate handle_remote_logging method, then just test that.

**Comment by enkore** [comment]
Fixed in d42f1e6 (never seen again), thanks
